dev/io.openliberty.grpc.1.0.internal.monitor/src/io/openliberty/grpc/internal/monitor/GrpcClientStats.java

```diff
@@ -0,0 +1,108 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.grpc.internal.monitor;
+
+import com.ibm.websphere.monitor.meters.Counter;
+import com.ibm.websphere.monitor.meters.Meter;
+
+import io.openliberty.grpc.GrpcClientStatsMXBean;
+
+/**
+ * Holds metrics used for client-side monitoring of gRPC services. </br>
+ * Statistic reported:
+ * <ul>
+ * <li>Total number of RPCs started on the client.
+ * <li>Total number of RPCs completed on the client, regardless of success or
+ * failure.
+ * <li>TODO Histogram of RPC response latency for completed RPCs, in seconds.
+ * <li>Total number of stream messages received from the server.
+ * <li>Total number of stream messages sent by the client.
+ * </ul>
+ */
+public class GrpcClientStats extends Meter implements GrpcClientStatsMXBean {
+	private final Counter rpcStarted;
+	private final Counter rpcCompleted;
+	private final Counter streamMessagesReceived;
+	private final Counter streamMessagesSent;
+
+	private final GrpcMethod method;
+
+	public GrpcClientStats(GrpcMethod method) {
+		this.method = method;
+
+		rpcStarted = new Counter();
+		rpcStarted.setDescription("This shows total number of RPCs started on the client");
```

Comment:
should these descriptions be NLS translated?

Reply:
I don't think so. I don't see other strings in NLS files.